jobs/furnishings/src/furnishings/config.py

```diff
@@ -82,6 +82,24 @@ class _Config:  # pylint: disable=too-few-public-methods
         name=DB_NAME,
     )
 
+    # BCLaws SFTP
+    BCLAWS_SFTP_STORAGE_DIRECTORY = os.getenv('BCLAWS_SFTP_STORAGE_DIRECTORY', None)
+    BCLAWS_SFTP_HOST = os.getenv('BCLAWS_SFTP_HOST', None)
+    BCLAWS_SFTP_PORT = os.getenv('BCLAWS_SFTP_PORT', None)
+    BCLAWS_SFTP_USERNAME = os.getenv('BCLAWS_SFTP_USERNAME', None)
+    BCLAWS_SFTP_PRIVATE_KEY_ALGORITHM = os.getenv('BCLAWS_SFTP_PRIVATE_KEY_ALGORITHM', 'ED25519')
+    BCLAWS_SFTP_PRIVATE_KEY_PASSPHRASE = os.getenv('BCLAWS_SFTP_PRIVATE_KEY_PASSPHRASE', None)
+    BCLAWS_SFTP_PRIVATE_KEY = os.getenv('BCLAWS_SFTP_PRIVATE_KEY', None)
```

Comment:
These will also need to be added to the dev's `.env` file. For local testing with the [sftp-server repo](https://github.com/leodube-aot/sftp-server), these are:

- `BCLAWS_SFTP_STORAGE_DIRECTORY`= The `SFTP_STORAGE_DIRECTORY` from `sftp-server.env`. Make sure the directory exists on the SFTP server.
- `BCLAWS_SFTP_HOST=localhost`
- `BCLAWS_SFTP_PORT=22`
- `BCLAWS_SFTP_USERNAME`= Any username
- `BCLAWS_SFTP_PRIVATE_KEY_ALGORITHM=ED25519`
- `BCLAWS_SFTP_PRIVATE_KEY_PASSPHRASE`= If a passphrase was used when generating the ssh keys
- `BCLAWS_SFTP_PRIVATE_KEY` = The private id key found in `keys/id_ed25519`